ystia/yorc
# Bugfix/gh 345 a4c premium https

# Pull Request description

## Description of the change

Fixed an issue bootstrapping a setup with an Alien4cloud premium version: the SSL configuration was unexpectedly not done.

### What I did

Added in tosca_types.zip bundle the script configuring Alien4Cloud updated by pull request https://github.com/alien4cloud/csar-public-library/pull/31 to support both premium and open source configurations.

### How to verify it

Bootstrap a setup using default Alien4Cloud settings (open source version).
Check Alien4Cloud UI is using https.

Bootstrap a setup specifying an Alien4Cloud premium dwnload URL.
Check Alien4Cloud UI is using https.


### Description for the changelog

Bootstrap using a premium version of Alien4Cloud fails to configure https/SSL ([GH-345](https://github.com/ystia/yorc/issues/345))

## Applicable Issues

https://github.com/ystia/yorc/issues/345
